src/System.Threading/ref/System.Threading.cs

```diff
@@ -31,22 +30,23 @@ public partial struct AsyncFlowControl : System.IDisposable
         public static bool operator !=(System.Threading.AsyncFlowControl a, System.Threading.AsyncFlowControl b) { throw null; }
         public void Undo() { }
     }
-    public sealed partial class AsyncLocal<T>
-    {
-        public AsyncLocal() { }
-        public AsyncLocal(System.Action<System.Threading.AsyncLocalValueChangedArgs<T>> valueChangedHandler) { }
-        public T Value { get { throw null; } set { } }
-    }
     public partial struct AsyncLocalValueChangedArgs<T>
     {
         private T _dummy;
+        private int _dummyPrimitive;
```

Comment:
Would this be considered a breaking change?

Reply:
Also, should this field be a bool given what's in the implementation?
https://github.com/dotnet/coreclr/blob/181680249af92aa45c8fb44a3ee865c7b61fd765/src/mscorlib/shared/System/Threading/AsyncLocal.cs#L98
```C#
public bool ThreadContextChanged { get; private set; }
```

Reply:
Yes it could break some folks similar to all the others we changed. This is fairly new API though I'm not too worried about.

Reply:
Yes it is OK for it to be an int.

Reply:
I'm a bit confused, why are there private fields in the ref?

Reply:
I see that dummy fields are there elsewhere as well but I don't understand why they need to be there

Reply:
> I see that dummy fields are there elsewhere as well but I don't understand why they need to be there

For context:
https://github.com/dotnet/corefx/issues/28870
https://github.com/dotnet/corefx/pull/28906